src/poker.js

```diff
@@ -1,10 +1,74 @@
-function winningPair() {}
+const values = {
+  0: 0,
+  2: 2,
+  3: 3,
+  4: 4,
+  5: 5,
+  6: 6,
+  7: 7,
+  8: 8,
+  9: 9,
+  10: 10,
+  J: 11,
+  Q: 12,
+  K: 13,
+  A: 14
+}
+
+function winningPair(firstPair, secondPair) {
+  if (
+    values[firstPair[0]] === values[firstPair[1]] &&
+    values[secondPair[0]] !== values[secondPair[1]]
+  )
+    return firstPair
+  else if (
+    values[firstPair[0]] !== values[firstPair[1]] &&
+    values[secondPair[0]] === values[secondPair[1]]
+  )
+    return secondPair
+  else if (
+    values[firstPair[0]] === values[firstPair[1]] &&
+    values[secondPair[0]] === values[secondPair[1]]
+  )
```

Comment:
Nice work on logic but this is particularly difficult to understand what all of the conditions mean. A nice improvement to make here would be to move the results of the evaluations into their own variables, e.g.

```js
const hand1IsPair = values[firstPair[0]] === values[firstPair[1]]
const hand2IsPair = values[secondPair[0]] === values[secondPair[1]]

if (hand1IsPair && hand2IsPair) {
```